tap_azure_tickets/__init__.py

```diff
@@ -0,0 +1,779 @@
+import os
+import json
+import functools
+from dateutil import parser
+import pytz
+import time
+import requests
+import re
+import singer
+import singer.bookmarks as bookmarks
+import singer.metrics as metrics
+
+
+from singer import metadata
+
+session = requests.Session()
+logger = singer.get_logger()
+
+# globals which hold data across streams
+workItemIds = []
+
+# map of Azure field names to more usable/readable field names
+FIELD_MAP = {
+    "System.Id": "id",
+    "System.AssignedTo": "assignedTo",
+    "Microsoft.VSTS.Common.ClosedBy": "closedBy",
```

Comment:
Gotta love Microsoft, this reminds me of IE 6, lol.

Reply:
Yeah if it weren't for the non-engineers at that company, their main website would be probably be https://vsts.common.public.marketing.microsoft.com/js-version=7.0-preview&$compat=ie